code/modules/atmospherics/machinery/pipes/pipes.dm

```diff
@@ -49,7 +49,7 @@
 		air_update_turf()
 
 /obj/machinery/atmospherics/pipe/return_air()
-	return parent.air
+	return parent?.air
```

Comment:
we do not want to silence this one, pipes are always supposed to have a parent

Reply:
Pipes dont have parent if it dont initialized.